src/index.tsx

```diff
@@ -0,0 +1,14 @@
+import React from "react";
+import ReactDOM from "react-dom";
+
+import { GlobalStyles } from "./GlobalStyles";
+
+import App from "./App";
+
+ReactDOM.render(
+  <React.StrictMode>
+    <GlobalStyles />
```

Comment:
You may, possibly, want to wrap your `<App />` into [`ThemeContext.Provider`](https://styled-components.com/docs/advanced#theming) to share some common styling parameters (palette, font settings, etc) across your entire app.

Reply:
Will look into that in the scope of the next HW since I didn't work with styled-components before. Thanks